sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -2835,6 +2958,18 @@ public partial class PeriodicModeProperties
         public int? BackupRetentionIntervalInHours { get { throw null; } set { } }
         public Azure.ResourceManager.CosmosDB.Models.CosmosDBBackupStorageRedundancy? BackupStorageRedundancy { get { throw null; } set { } }
     }
+    public partial class Privilege
+    {
+        public Privilege() { }
+        public System.Collections.Generic.IList<string> Actions { get { throw null; } }
+        public Azure.ResourceManager.CosmosDB.Models.PrivilegeResource Resource { get { throw null; } set { } }
+    }
+    public partial class PrivilegeResource
```

Comment:
Rename to `PrivilegeResourceInfo` as `Resource` suffix has a special meaning in .NET Azure SDK.